### PR TITLE
[Task] Update webpack

### LINKED ISSUE #201: [Improvement] Upgrade symfony/webpack-encore-bundle
### Improvement description

...

### PR DESCRIPTION
**Pull request by mattamon**
Resolves #201 
Update to 1.17 possible
Restricted due to pimcore/admin-ui-classic-bundle

```
 - Root composer.json requires pimcore/admin-ui-classic-bundle * -> satisfiable by pimcore/admin-ui-classic-bundle[dev-722-fix-manual-sorting, 1.6.x-dev].
 - pimcore/admin-ui-classic-bundle 1.6.x-dev requires symfony/webpack-encore-bundle ^1.13.2 -> satisfiable by symfony/webpack-encore-bundle[v1.13.2, ..., v1.17.2].

```

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -20,7 +20,7 @@
     "pimcore/personalization-bundle": "^1.0",
     "pimcore/google-marketing-bundle": "^1.0",
     "knplabs/knp-paginator-bundle": "^6.0.0",
-    "symfony/webpack-encore-bundle": "^1.13.2",
+    "symfony/webpack-encore-bundle": "^1.17",
```
Comment (blankse) [inline composer.json]: @mattamon I think we could support both versions. Also in admin-ui-classic-bundle. WDYT?
```suggestion
    "symfony/webpack-encore-bundle": "^1.13.2 || ^2.0",
```

Comment (mattamon) [reply composer.json]: yeah but I would at least already set it to 1.17 so like ^1.17 || ^2.0

**Comment by blankse** [comment]
@mattamon `symfony/webpack-encore-bundle < 1.13.2` can't be installed. So it makes no sense to set it lower.

File: composer.json [hunk 2]
```diff
@@ -20,7 +20,7 @@
     "pimcore/personalization-bundle": "^1.0",
     "pimcore/google-marketing-bundle": "^1.0",
     "knplabs/knp-paginator-bundle": "^6.0.0",
-    "symfony/webpack-encore-bundle": "^1.13.2",
+    "symfony/webpack-encore-bundle": "^1.7 || ^2.0",
```
Comment (jdreesen) [inline composer.json]: It was probably a typo.

```suggestion
    "symfony/webpack-encore-bundle": "^1.17 || ^2.0",
```

**Comment by mattamon** [comment]
> @mattamon `symfony/webpack-encore-bundle < 1.13.2` can't be installed. So it makes no sense to set it lower.

yeah sorry it was a typo

**Comment by kingjia90** [comment]
Mhhh, the tests are false positive, because it's still installing the v1, due to personalization bundle which requires an admin ui bundle that doesn't have https://github.com/pimcore/admin-ui-classic-bundle/pull/770 yet

**Pull request merged**